snowshu/adapters/target_adapters/base_local_target_adapter.py

```diff
@@ -160,9 +160,7 @@ def create_all_database_extensions(self):
         """Create all required database extensions for the target adapter."""
 
     @abstractmethod
-    def initialize_replica(
-        self, source_adapter_name: str, incremental_image: str = None
-    ) -> None:
+    def initialize_replica(self, source_adapter_name: str, **kwargs) -> None:
```

Comment:
If we put `incremental_image` into kwargs, let's change the docstring accordingly